eager_solid_other/launch/gazebo.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="ns" default="multisense_s21"  doc="Namespace of the solid." />
```

Comment:
`multisense_s21` namespace?

Reply:
Well, it is just the default value for the namespace. The true namespace value is specified by the eager_env, e.g. eager_env/objects/sensors/ms21

Reply:
Oh I now see that this is the launch file of the solid, sorry about that. I will fix it.